.github/workflows/auto_community_labeler.yml

```diff
@@ -0,0 +1,27 @@
+
+name: Auto Community Labeler
+
+on:
+  pull_request:
+  issues:
+
+jobs:
+  auto_PR_labeler:
+    name: Auto PR Community Labeler
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'pull_request' && github.event.pull_request.author_association != 'MEMBER' && github.event.pull_request.author_association != 'OWNER' }}
```

Comment:
Shouldn't the or statement `||` then be between brackets to differentiate it from the other `&&`?
E.g. like so:
```suggestion
    if: ${{ github.event_name == 'pull_request' && (github.event.pull_request.author_association != 'MEMBER' || github.event.pull_request.author_association != 'OWNER') }}
```
No idea whether that's the case?

Reply:
Also, couldn't these be merged into one `if` action so that if `github.event_name` is in `["pull_request", "issues"]`?

Reply:
tbh, I'm not very familiar (still learning) with github actions and `.yml` syntax.
I'm not sure if `github.event_name == 'issues'` and `github.event_name == 'pull_request'` are redundant

Reply:
I have never written any github actions - I just felt it was weird there's code that's saying:
```
on:
  pull_request:
  issues:
```

That it would ever run for anything else. Let me do a quick google search :)